src/components/buttons/MiddleButtons.js

```diff
@@ -0,0 +1,50 @@
+import {Color, Typography} from '../../themes';
+import {ColorVariant} from '../../themes/color';
+import ButtonItem from './buttonItem/ButtonItem';
+import {StyleSheet} from 'react-native';
+
+export default function SmallButtons(props) {
+  const {
+    content,
+    message,
+    colorVariant = ColorVariant.primary,
+    typographyVariant = Typography.body.medium,
+    children,
```

Comment:
props "children" will be useless when extract from "props" -> otherProps  will no longer contain "children" props

Reply:
sorry but would you please explain more detail about this comment? What is the difference between children and otherProps ?